client/my-sites/site-settings/section-export.jsx

```diff
@@ -10,24 +10,32 @@ import { localize } from 'i18n-calypso';
  */
 import EmptyContent from 'components/empty-content';
 import ExporterContainer from 'my-sites/exporter';
-import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import {
+	getSelectedSite,
+	getSelectedSiteId,
+	getSelectedSiteSlug,
+} from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
 
-const SiteSettingsExport = ( { isJetpack, site, translate } ) => {
-	if ( isJetpack ) {
-		return (
-			<EmptyContent
+const SiteSettingsExport = ( { isJetpack, site, siteSlug, translate } ) => {
+	return (
+		<Main>
+			<HeaderCake backHref={ '/settings/general/' + siteSlug }>
+				<h1>{ translate( 'Import' ) }</h1>
+			</HeaderCake>
+			{ isJetpack && <EmptyContent
 				illustration="/calypso/images/drake/drake-jetpack.svg"
 				title={ translate( 'Want to export your site?' ) }
 				line={ translate( 'Visit your site\'s wp-admin for all your import and export needs.' ) }
 				action={ translate( 'Export %(siteTitle)s', { args: { siteTitle: site.title } } ) }
 				actionURL={ site.options.admin_url + 'export.php' }
 				actionTarget="_blank"
-			/>
-		);
-	}
-
-	return <ExporterContainer />;
+			/> }
+			{ ! isJetpack && <ExporterContainer /> }
```

Comment:
I'd suggest being more concrete here and alter this one to:

```
{ isJetpack === false && <ExporterContainer /> }
```
because right now the export container can appear for a short while for a Jetpack sites if sites are not loaded. To reproduce:

* Clear your Redux state
* Go to `/settings/export/$site` where `$site` is one of your Jetpack sites.
* You're seeing the `ExporterContainer` while the sites are loading (which I think can be confusing to the user).
* After sites are loaded, you are seeing the `<EmptyContent />`.